dotnet/src/Connectors/Connectors.AI.Oobabooga/TextCompletion/TextCompletionResult.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.AI.TextCompletion;
+using Microsoft.SemanticKernel.Orchestration;
+
+namespace Microsoft.SemanticKernel.Connectors.AI.Oobabooga.TextCompletion;
+
+internal sealed class TextCompletionResult : ITextCompletionResult
```

Comment:
I did add summary comments to those internal result classes, but that's about it, because there is actually no comment in any of the existing classes implementing the same interfaces. I don't want to set the standard higher than it is for paid contributors. Hopefully you're OK with that.

Reply:
No need to document `internal` classes, methods and properties (at least for now). Just the `public` ones require documentation. In case all public classes, methods and properties are documented, feel free to resolve this comment, thanks.